js/registrationForm.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <script src="registrationForm.js"></script>
+    <title>Registration form</title>
+</head>
+<body>
+    <form action="registrationForm.php" onsubmit="return validate()" method="POST">
+        <label for="username">Username:</label>
```

Comment:
You may want to add validations on the html inputs. Giving feedback to the user as early as possible is always better and this will save him time. You could check here for a reference: https://github.com/martinely95/WEB2020-php/blob/master/ex5/index.html